src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
create type Shape = 'triangle' | 'circle' | 'rectangle' and type Color = 'red' | 'green' | 'blue';
Now you can reusing it in other places.